test/functionalTest/cases/950_GET_v2_entity/GET_v2_entity_not_found.test

```diff
@@ -0,0 +1,277 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
The former test was already testing for non existence. I think it would be better not to repeat tests ... or focus them in testing only one thing